src/api/users/test/e2e/users.test.js

```diff
@@ -101,6 +103,87 @@ describe('GET REQUESTS', () => {
     ]);
   });
 
+  test('Accepted - Get all users (using params), only two users should be returned', async () => {
+    // create 3 users
+    const galileo1 = await createUser({ id: 10001 });
+    const galileo2 = await createUser({ id: 10002 });
+    const galileo3 = await createUser({ id: 10003 });
+
+    expect(galileo1.response.status).toBe(201);
+    expect(galileo2.response.status).toBe(201);
+    expect(galileo3.response.status).toBe(201);
+
+    // request 2 users per page on page 1, only two users should be returned
+    const response = await getUsersParams(2, 1);
+    expect(response.status).toBe(200);
+    expect(response.body.length).toBe(2);
+    expect(response.body).toEqual([
+      {
+        id: 10001,
+        firstName: 'Galileo',
+        lastName: 'Galilei',
+        displayName: 'Galileo Galilei',
+        isAdmin: true,
+        isFlagged: true,
+        feeds: ['https://dev.to/feed/galileogalilei'],
+        github: {
+          username: 'galileogalilei',
+          avatarUrl:
+            'https://avatars.githubusercontent.com/u/7242003?s=460&u=733c50a2f50ba297ed30f6b5921a511c2f43bfee&v=4',
+        },
+      },
+      {
+        id: 10002,
+        firstName: 'Galileo',
+        lastName: 'Galilei',
+        displayName: 'Galileo Galilei',
+        isAdmin: true,
+        isFlagged: true,
+        feeds: ['https://dev.to/feed/galileogalilei'],
+        github: {
+          username: 'galileogalilei',
+          avatarUrl:
+            'https://avatars.githubusercontent.com/u/7242003?s=460&u=733c50a2f50ba297ed30f6b5921a511c2f43bfee&v=4',
+        },
+      },
+    ]);
+  });
+
+  test('Accepted - Get 20 uncreated users (using params), receive empty array', async () => {
```

Comment:
Add a test with only `per_page` and only `page`, make sure defaults get applied as you expect.